Update Res

zzauguste opened the pull request.

augbac committed:
Update Res

augbac committed:
Update Cap

augbac committed:
Update Cap

augbac committed:
Update Capacitors

Aleksandrovas commented:
Pakeitimai nėra priimti, įveskite visą reikiamą informaciją. Platesnė informacija Moodle sistemoje Komponentų bibliotekos.pdf

augbac committed:
Capacitors_update

augbac committed:
Update Res and Cap

New resistors:
AC0603FR-07100RL
CR0805-FX-1000ELF
RC1206FR-07100RL

new capacitors:
C1206C103F5GACTU
C0603C103F3GECTU
C0805X103F5GACTU

Aleksandrovas committed:
Merge branch 'KTU_Lib_master' into pr/6
